packages/admin/src/mui/useComponentThemeProps.tsx

```diff
@@ -0,0 +1,8 @@
+import { useTheme } from "@material-ui/core";
+import { ComponentsProps } from "@material-ui/core/styles/props";
+
+export function useComponentThemeProps<T>(name: keyof ComponentsProps): T | undefined {
```

Comment:
What do we need `useComponentThemeProps ` for?
To access the default props of a component which are defined in the theme?

So this is supposed to be a common pattern for components to access their default props?

Reply:
is that minimal hook worth it?

Reply:
yes i think we should stick on this minimal hook. I have seen some custom implementations in some MR and i think this hook makes this easier and more stable (Usages: https://github.com/vivid-planet/comet-admin/pull/338/files#diff-7774cd41820ae34883dcdd5870be36c80ff65effa6e66ecdf208408462dffddaR102). 

More infos to this hook see https://github.com/vivid-planet/comet-admin/pull/369#discussion_r627215102

Reply:
i think there is also the possibility with type inference to get the return value (T) automtically infered by the passed name, but i did not got this working, maybe @rabengraph can help?

Reply:
@manuelblum ...on the way! https://github.com/vivid-planet/comet-admin/pull/372